tests/contest/runtimetest/src/tests.rs

```diff
@@ -820,3 +820,41 @@ pub fn validate_fd_control(_spec: &Spec) {
         eprintln!("mismatched fds inside container! {:?}", fd_details);
     }
 }
+
+pub fn validate_masked_paths(spec: &Spec) {
+    let linux = spec.linux().as_ref().unwrap();
+    let masked_paths = match linux.masked_paths() {
+        Some(p) => p,
+        None => {
+            eprintln!("in masked paths, expected some masked paths to be set, found none");
+            return;
+        }
+    };
+
+    for path_str in masked_paths {
```

Comment:
How about  handling `path_str` as Path, not &str?
```suggestion
    for path in masked_paths.iter().map(Path::new) {
```

Following the above change, we'll amend some functions, like:
```
pub fn test_read_access<P: AsRef<Path>>(path: P) -> Result<Readable, std::io::Error> {
```

Reply:
Most of the function arguments in `tests/contest/runtimetest/src/utils.rs`, including `test_read_access`, are handled as `&str`. Should they still be modified? If so, should the change be applied to all of them?

Reply:
You don't need to take care of other functions in this PR. I'm glad to address them in another PR if possible 🙏